plugins.d/alarm-notify.sh

```diff
@@ -975,6 +976,37 @@ send_email() {
     return 1
 }
 
+# -----------------------------------------------------------------------------
+# Custom.  Send to Alexa
+
+send_alexa() {
+    local httpcode sent=0
+
+    if [ "${SEND_ALEXA}" = "YES" -a ! -z "${ACCESSCODES}" ]
```

Comment:
This should be written as `if [ "${SEND_ALEXA}" = "YES" ] && [ ! -z "${ACCESSCODES}" ]`. I know all other places use notation specified by you, but this will soon change as we are in the process of linting our bash scripts using shellcheck.